stable/postgresql/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: postgresql
-version: 3.1.3
+version: 3.1.4
```

Comment:
someone already bumped to this version in other PR. Can you increase to 3.1.5?

Reply:
Actually, it should 3.2.1 now! 😬

Reply:
Ahh this chart is fast moving. its actually  3.5.1 now. :)